### PR TITLE
netdev: bond - add support for peer_notif_delay 

### PR DESCRIPTION
**Pull request by ssahani**

### CODE REVIEW

File: man/systemd.netdev.xml [hunk 1]
```diff
@@ -2182,6 +2182,19 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>PeerNotifyDelaySec=</varname></term>
+        <listitem>
+          <para>Specifies the number of seconds the delay between each peer notification
+          (gratuitous ARP and unsolicited IPv6 Neighbor Advertisement) when they are issued
+          after a failover event. This delay should be a multiple of the MII link monitor
+          interval (miimon). The valid range is 0...300. The default value is 0,
```
Comment (poettering) [inline man/systemd.netdev.xml]: say `300s`, not just `300`

File: src/network/netdev/bond.c [hunk 2]
```diff
@@ -88,6 +88,12 @@ static int netdev_bond_fill_message_create(NetDev *netdev, Link *link, sd_netlin
                         return r;
         }
 
+        if (b->peer_notify_delay != 0) {
+                r = sd_netlink_message_append_u32(m, IFLA_BOND_PEER_NOTIF_DELAY, b->peer_notify_delay / USEC_PER_MSEC);
```
Comment (poettering) [inline src/network/netdev/bond.c:92]: should this use DIV_ROUND_UP()?

(just asking, i have no idea what would be better here: rounding down or rounding up)

File: src/network/netdev/bond.h [hunk 3]
```diff
@@ -29,6 +29,7 @@ typedef struct Bond {
         unsigned packets_per_slave;
         unsigned num_grat_arp;
         unsigned min_links;
+        unsigned peer_notify_delay;
```
Comment (poettering) [inline src/network/netdev/bond.h]: must be usec_t

File: src/network/netdev/bond.c (hunk 2 above)
Comment (ssahani) [reply src/network/netdev/bond.c:92]: we have `downdelay` `updelay` and more same as `peer_notify_delay`. I think it's not required.

**Comment by ssahani** [comment]
Updated thanks for the review !

**Review by poettering — APPROVED** [review]

**Review by yuwata — APPROVED** [review]

**Pull request merged**